pkg/channel/http/http_channel.go

```diff
@@ -231,34 +231,26 @@ func (h *Channel) invokeMethodV1(ctx context.Context, req *invokev1.InvokeMethod
 	diag.DefaultHTTPMonitoring.ClientRequestStarted(ctx, int64(len(req.Message().GetData().GetValue())))
 	startRequest := time.Now()
 
-	var resp *http.Response
-	if len(h.pipeline.Handlers) > 0 {
-		// Exec pipeline only if at least one handler is specified
-		rw := &RWRecorder{
-			W: &bytes.Buffer{},
-		}
-		execPipeline := h.pipeline.Apply(http.HandlerFunc(func(wr http.ResponseWriter, r *http.Request) {
-			// Send request to user application
-			// (Body is closed below, but linter isn't detecting that)
-			//nolint:bodyclose
-			clientResp, clientErr := h.client.Do(r)
-			if clientResp != nil {
-				copyHeader(wr.Header(), clientResp.Header)
-				wr.WriteHeader(clientResp.StatusCode)
-				_, _ = io.Copy(wr, clientResp.Body)
-			}
-			if clientErr != nil {
-				err = clientErr
-			}
-		}))
-		execPipeline.ServeHTTP(rw, channelReq)
-		resp = rw.Result() //nolint:bodyclose
-	} else {
+	// Exec pipeline only if at least one handler is specified
```

Comment:
With `if len(h.pipeline.Handlers) > 0 {` removed, is this still true?

Reply:
Removed